k9mail/src/main/java/com/fsck/k9/activity/MessageCompose.java

```diff
@@ -1939,7 +1939,28 @@ private void showOrHideQuotedText(QuotedTextMode mode) {
             }
         }
     }
-
+    
+    private void askBeforeDiscard(){
```

Comment:
This method still contains quite a few code style issues. Missing spaces, wrong indentation, lines that are too long.

Importing the settings to Android Studio, selecting all text of the method, then using "reformat code" should do the right thing. See https://github.com/k9mail/k-9/wiki/CodeStyle